mmhuman3d/data/data_converters/gta_human.py

```diff
@@ -0,0 +1,293 @@
+import glob
+import os
+import pickle
+from typing import Tuple
+
+import numpy as np
+import torch
+from tqdm import tqdm
+
+from mmhuman3d.core.conventions.keypoints_mapping import (
+    convert_kps,
+    get_keypoint_idx,
+)
+from mmhuman3d.data.data_structures.human_data import HumanData
+from mmhuman3d.models.builder import build_body_model
+from .base_converter import BaseConverter
+from .builder import DATA_CONVERTERS
+
+# TODO:
+# 1. camera parameters
+# 2. root align using mid-point of hips
+
+
+def perspective_projection(points: torch.Tensor, focal_length: float,
+                           camera_center: Tuple):
+    """This function computes the perspective projection of a set of points.
+
+    Input:
+        points (bs, N, 3): 3D points
+        focal_length (bs,) or scalar: Focal length
+        camera_center (bs, 2): Camera center
+    """
+    batch_size = points.shape[0]
+    K = torch.zeros([batch_size, 3, 3], device=points.device)
+    K[:, 0, 0] = focal_length
+    K[:, 1, 1] = focal_length
+    K[:, 2, 2] = 1.
+
+    K[:, 0, -1] = camera_center[0]
+    K[:, 1, -1] = camera_center[1]
+
+    points = torch.transpose(points, 1, 2)  # (bs, 3, N)
+    projected_points = torch.matmul(K, points)  # (bs, 3, N)
+    projected_points = torch.transpose(projected_points, 1, 2)  # (bs, N, 3)
+    projected_points = projected_points[:, :, :
+                                        2] / projected_points[:, :,
+                                                              2].unsqueeze(
+                                                                  -1
+                                                              )  # (bs, N, 2)
```

Comment:
p.s. small issue but the formatting looks abit off here

You might also want to check with @JasonBoy1 if there's an existing function for perspective projection-  https://github.com/open-mmlab/mmhuman3d/blob/f107203714f9627a9308d4515d35ab8fbd0074a4/mmhuman3d/utils/geometry.py#L229-L257 or 
https://github.com/open-mmlab/mmhuman3d/blob/f107203714f9627a9308d4515d35ab8fbd0074a4/mmhuman3d/core/parametric_model/smplify.py#L14-L52

Reply:
It is changed to the standard perspective projection used in the codebase.

Reply:
great